dockers/base-conda/Dockerfile

```diff
@@ -29,7 +29,7 @@ FROM nvidia/cuda:${CUDA_VERSION}-cudnn${CUDNN_VERSION}-devel-ubuntu18.04
 ARG PYTHON_VERSION=3.7
 ARG PYTORCH_VERSION=1.6
 ARG PYTORCH_CHANNEL=pytorch
-ARG CONDA_VERSION=4.7.12
+ARG CONDA_VERSION=latest
```

Comment:
this was for reason, some 1.8 version did not pass failed status, so even tests failed it exited in CI as pass
So at least add fix version and check that with `exit(1)` the CI fails...

Reply:
Fixed in 4.9.1, but can't found on repo.anaconda.com/miniconda, so back to old version.